utilities/node_activator/README.md

```diff
@@ -6,7 +6,29 @@ A module to activate VM nodes.
 
 ## Installation
 
-TBD
+In order to use `NodeActivator`, you will need Elixir installed. Then create an Elixir project, whose name is `my_app`, for example, via the `mix` build tool:
```

Comment:
I think it can be clearer if we separate it from the sentence. `whose name is ...` can be confusing. 

```suggestion
In order to use `NodeActivator`, you will need Elixir installed. Then create an Elixir project via the `mix` build tool. For example, if your project is `my_app`, you may run the following command.
```

Or what about simplify it like this?

```suggestion
In order to use `NodeActivator`, you will need Elixir installed. Then create an Elixir project via the `mix` build tool:
```

If the target audience is non-Elixir programmers, we might need extra explanation. Probably they will figure out what `my_app` indicates in the example.